app/views/content/non-uk-teachers/visas-for-non-UK-trainees.md

```diff
@@ -0,0 +1,257 @@
+---
+title: "Apply for your visa to train to teach in England"
+description: |-
+  Learn more about types of visa non-UK citizens will need to study or work in England, and how to apply.
+related_content:
```

Comment:
Should the main page not be included? https://getintoteaching.education.gov.uk/non-uk-teachers/train-to-teach-in-england-as-an-international-student